zingoconfig/src/lib.rs

```diff
@@ -30,6 +30,15 @@ use zcash_primitives::consensus::{
 /// TODO: Add Doc Comment Here!
 pub const DEVELOPER_DONATION_ADDRESS: &str = "u1w47nzy4z5g9zvm4h2s4ztpl8vrdmlclqz5sz02742zs5j3tz232u4safvv9kplg7g06wpk5fx0k0rx3r9gg4qk6nkg4c0ey57l0dyxtatqf8403xat7vyge7mmen7zwjcgvryg22khtg3327s6mqqkxnpwlnrt27kxhwg37qys2kpn2d2jl2zkk44l7j7hq9az82594u3qaescr3c9v";
 /// TODO: Add Doc Comment Here!
+pub const ZENNIES_FOR_ZINGO_DONATION_ADDRESS: &str = "u1p32nu0pgev5cr0u6t4ja9lcn29kaw37xch8nyglwvp7grl07f72c46hxvw0u3q58ks43ntg324fmulc2xqf4xl3pv42s232m25vaukp05s6av9z76s3evsstax4u6f5g7tql5yqwuks9t4ef6vdayfmrsymenqtshgxzj59hdydzygesqa7pdpw463hu7afqf4an29m69kfasdwr494";
+/// TODO: Add Doc Comment Here!
+pub const ZENNIES_FOR_ZINGO_REGTEST_ADDRESS: &str = "uregtest14emvr2anyul683p43d0ck55c04r65ld6f0shetcn77z8j7m64hm4ku3wguf60s75f0g3s7r7g89z22f3ff5tsfgr45efj4pe2gyg5krqp5vvl3afu0280zp9ru2379zat5y6nkqkwjxsvpq5900kchcgzaw8v8z3ggt5yymnuj9hymtv3p533fcrk2wnj48g5vg42vle08c2xtanq0e";
+/// TODO: Add Doc Comment Here!
+pub const ZENNIES_FOR_ZINGO_AMOUNT: u64 = 1_000_000;
+/// TODO: Add Doc Comment Here!
+pub const ZENNIES_FOR_ZINGO_MEMO_BYTES: &[u8] =
```

Comment:
I am -1 on this memo.   What is the purpose?

Reply:
I know this has been discussed in other channels but the reason is so the user has an "on chain" reference to which transactions included zingo tips. I think we have now decided to resolve this by adding this address to the default address book in zingo-mobile but this solution doesnt work for zingo-cli until we add address book. i am happy with this solution for now, the only other option i have is we have a unviersal memo no matter what language is selected saying "Zenny for Zingo!"